### PR TITLE
dependency-updates

### LINKED ISSUE #52: Links in Readme broken
Have a look at the readme, the following links are broken:
- user-guide
- vs-doc
- vs-api

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #52

### CODE REVIEW

**Review by pj-spoelders — DISMISSED** [review]

**Review by pj-spoelders — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -29,19 +26,21 @@ If you want to set up a Virtual Schema for a different database system, please h
...
 * [Changelog](doc/changes/changelog.md)
 * [Dependencies](dependencies.md)
 
 Find all the documentation in the [Virtual Schemas project][vs-doc].
 
 ## Information for Developers
 
-* [Virtual Schema API Documentation][vs-api]
\ No newline at end of file
+* [Virtual Schema API Documentation][vs-api]
```
Comment (AnastasiiaSergienko) [inline README.md:44]: This doesn't look like a valid link

File: README.md [hunk 2]
```diff
@@ -29,19 +26,21 @@ If you want to set up a Virtual Schema for a different database system, please h
...
+It takes the name of a connection definition in case you are using `IMPORT FROM EXA`. For details please consult the [user guide][virtual-schemas-user-guide].
 
 ## Table of Contents
 
 ### Information for Users
 
-* [Exasol dialect](doc/dialects/exasol.md)
-* [User Guide][user-guide]
+* [Virtual Schemas User Guide][virtual-schemas-user-guide]
+* [Generic Virtual Schema User Guide](doc/dialects/exasol.md)
```
Comment (AnastasiiaSergienko) [inline README.md:36]:
```suggestion
* [Exasol Dialect User Guide](doc/dialects/exasol.md)
```

File: README.md [hunk 3]
```diff
@@ -29,19 +26,21 @@ If you want to set up a Virtual Schema for a different database system, please h
 
 Starting with version 4.0.0 of the Exasol SQL dialect, the new `EXA_CONNECTION` property replaces the `EXA_CONNECTION_STRING` property, for improved security.
 
-It takes the name of a connection definition in case you are using `IMPORT FROM EXA`. For details please consult the [user guide][user-guide].
+It takes the name of a connection definition in case you are using `IMPORT FROM EXA`. For details please consult the [user guide][virtual-schemas-user-guide].
```
Comment (AnastasiiaSergienko) [inline README.md:29]: The link should be to doc/dialects/exasol.md

File: pom.xml [hunk 4]
```diff
@@ -199,26 +192,49 @@
...
+                <version>0.10.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>verify</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <modules>
```
Comment (AnastasiiaSergienko) [inline pom.xml:226]: Does it have a module for maven release?

**Review by AnastasiiaSergienko — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (jakobbraun) [reply pom.xml:226]: yes

**Review by AnastasiiaSergienko — APPROVED** [review]

**Pull request merged**